accounts-journey/presentation/ui/accounts_list/AccountsListViewController.swift

```diff
@@ -81,9 +68,9 @@ final class AccountsListViewController: UIViewController {
     
     private func setupLayout() {
         accountsListTableView.snp.makeConstraints { make in
-            make.leading.trailing
+            make.leading.trailing.top
                 .equalToSuperview().inset(DesignSystem.shared.spacer.md)
-            make.top.bottom.equalTo(view.safeAreaLayoutGuide)
+            make.bottom.equalTo(view.safeAreaLayoutGuide)
```

Comment:
not really a comment for this PR, but werent we trying to get rid of SnapKit as Backbase? i feel like we shouldnt have it for the golden sample app if thats the case

Reply:
cc @DevAgani @tibor-is-back

Reply:
I had discussion regarding this in the mobile leadership, we will change the Golden Sample App but not within this PR.